### DISCUSSION TITLE
How to get C++ API document?

### DISCUSSION THREAD

**Discussion by IGabriel** [onnx/onnx#5454] [Q&A]
I am looking for the ONNX C++ reference.

On the home page of onnx (https://onnx.ai/), I get the python api reference only.
And in the release note, it mentions the Python API only.
Hom can I get the ONNX C++ reference?

Any help is appreciated!

**Comment by artificalaudio** [comment]
@IGabriel here you go: https://onnxruntime.ai/docs/api/c/
